### PR TITLE
Hotfix+Refactoring

### PR DESCRIPTION
**Pull request by HarishTeens**
This PR includes

- Fix errors after Deploying
- Separated Access Level CRUD operation from Update Club
- Ensure all API are consistent in terms of responses and error codes
- Complete Add Club API for clubs onboarding with all necessary properties

below is the finalised schema for clubs.
```
{
  clubName: String,
  memberAccess: [AcessLevel],
  facAd: String,
  logo: String,
  events: [Event],
  society: String,
  domain: String,
  links: String,	
  backgroundColor: String, 
  contactInfo: [{ 
    name: String,
    designation: String,
    mobileNo: String,
    email: String
}
```

Fixes #
#61

### CODE REVIEW

**Comment by DesignrKnight** [comment]
There was supposed to be a description for clubs na?

**Comment by DesignrKnight** [comment]
links is supposed to be an array of strings

**Comment by DesignrKnight** [comment]
How are we handling dark and light set both with background colour and logo as individual field?

**Comment by HarishTeens** [comment]
Hey @DesignrKnight ,
1. missed description property for clubs, will add that
2. presently(scope of this PR) to keep it simple, we'll be using a JSON
3. couldnt get you?

**Comment by HarishTeens** [comment]
@roshankshaw check it out. see if its alrighty. we'll deploy and test once in production. Once this is over, we are good to move on to events and venues.

File: sources/datasources/stories.js [hunk 1]
```diff
@@ -53,17 +55,21 @@ class StoryAPI extends DataSource {
...
-			const eventId = story.event;
-			const foundEvent = await Events.findById(eventId);
-            if(foundEvent==undefined){
-                return new ApolloError("Event Not Found");
-            }
-            createdStory.event = foundEvent._id;
+            const eventId = story.event;
+            try{
+                const foundEvent = await Events.findById(eventId);
+                if(foundEvent==undefined){
```
Comment (roshankshaw) [inline sources/datasources/stories.js:66]: This should be outside try-catch block.

File: sources/schema/club.js [hunk 2]
```diff
@@ -54,12 +59,12 @@ const queries = `
...
     clubByName(name:String):Club
     clubById(id:String):Club
+    deleteClub(id:ID!):ResponseResult
 `;
 
 const mutations = `
-    addClub(club:ClubInputType):Club,
-    updateClub(id:ID,club:ClubInputType):Club,
-    deleteClub(id:ID):Club,
+    addClub(club:ClubInputType):ClubResult,
```
Comment (roshankshaw) [inline sources/schema/club.js:66]: Mutation should return Responseresult?

Comment (HarishTeens) [reply sources/schema/club.js:66]: yes yes, thanks for pointing it out.

Comment (HarishTeens) [reply sources/schema/club.js:66]: so there are two kinds of mutations,
1. which returns a result object
2. which just returns a success message

addClub returns the created club object, but mutations like deleteClub only returns a success message.

File: sources/datasources/stories.js (hunk 1 above)
Comment (HarishTeens) [reply sources/datasources/stories.js]: moved it out.

**Pull request merged**